src/components/home/Home.js

```diff
@@ -0,0 +1,95 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchCountries } from '../../redux/countries/countries';
+import Icon from '../Icon';
+import formatNumber from '../../utilities/formatNumber';
+import './Home.css';
+import virus from '../../assets/virus.png';
+import map from '../../assets/southamerica.png';
+
+const Item = ({ confirmed, name }) => (
+  <div className="Home-item-content">
+    <div className="Home-item-icon">
+      <Icon name="arrow_circle_right" />
+    </div>
+    <div className="Home-item-top">
+      <img src={virus} alt="" className="Home-item-image" />
+    </div>
+    <div className="Home-item-bottom">
+      <h4 className="App-title">{name}</h4>
+      <p className="App-subtitle">{formatNumber(confirmed)}</p>
+    </div>
+  </div>
+);
+
+const Grid = ({ items = [] }) => (
+  <ul className="Home-grid">
+    {items.map(({ name, confirmed }) => (
+      <li key={name} className="Home-grid-item">
+        <Link to={`/country/${name}`}>
+          <Item confirmed={confirmed} name={name} />
+        </Link>
+      </li>
+    ))}
+  </ul>
+);
```

Comment:
- [ ] It will be great to implement the filter as requested by the first reviewer. Great work explaining on the pull request how the filter functionality is working in your project. However, for this functionality to be considered complete it should accomplish these points:

1. Your filter component should be a separate stateless component. This is a hard requirement for this project. Instead of just filtering the data in the home component have a separate component that is responsible for this functionality.

Reply:
Task completed ✔